src/apple/macos/ffi.rs

```diff
@@ -8,77 +8,80 @@ use core_foundation_sys::base::CFAllocatorRef;
 use core_foundation_sys::dictionary::CFMutableDictionaryRef;
 use core_foundation_sys::string::{CFStringEncoding, CFStringRef};
 
-#[cfg(not(feature = "apple-app-store"))]
-use libc::c_int;
-use libc::{c_char, c_void, mach_port_t, size_t};
+use cfg_if::cfg_if;
 
-pub(crate) use crate::sys::ffi::*;
-
-extern "C" {
-    #[cfg(not(feature = "apple-app-store"))]
-    pub fn mach_absolute_time() -> u64;
-
-    // The proc_* PID functions are internal Apple APIs which are not
-    // allowed in App store releases as Apple blocks any binary using them.
-
-    #[cfg(not(feature = "apple-app-store"))]
-    pub fn proc_pidinfo(
-        pid: c_int,
-        flavor: c_int,
-        arg: u64,
-        buffer: *mut c_void,
-        buffersize: c_int,
-    ) -> c_int;
-
-    #[cfg(not(feature = "apple-app-store"))]
-    pub fn proc_listallpids(buffer: *mut c_void, buffersize: c_int) -> c_int;
-    //pub fn proc_listpids(kind: u32, x: u32, buffer: *mut c_void, buffersize: c_int) -> c_int;
-    //pub fn proc_name(pid: c_int, buffer: *mut c_void, buffersize: u32) -> c_int;
-    //pub fn proc_regionfilename(pid: c_int, address: u64, buffer: *mut c_void,
-    //                           buffersize: u32) -> c_int;
-
-    #[cfg(not(feature = "apple-app-store"))]
-    pub fn proc_pidpath(pid: c_int, buffer: *mut c_void, buffersize: u32) -> c_int;
-
-    #[cfg(not(feature = "apple-app-store"))]
-    pub fn proc_pid_rusage(pid: c_int, flavor: c_int, buffer: *mut c_void) -> c_int;
-
-    // IOKit is only available on MacOS: https://developer.apple.com/documentation/iokit
-
-    pub fn IOMasterPort(a: i32, b: *mut mach_port_t) -> i32;
-
-    pub fn IOServiceMatching(a: *const c_char) -> *mut c_void;
-
-    pub fn IOServiceGetMatchingServices(
-        a: mach_port_t,
-        b: *mut c_void,
-        c: *mut io_iterator_t,
-    ) -> i32;
-
-    pub fn IOIteratorNext(iterator: io_iterator_t) -> io_object_t;
-
-    pub fn IOObjectRelease(obj: io_object_t) -> i32;
-
-    pub fn IOServiceOpen(device: io_object_t, a: u32, t: u32, x: *mut io_connect_t) -> i32;
+use libc::{c_char, c_void};
+#[cfg(not(feature = "apple-sandbox"))]
+use libc::{c_int, mach_port_t, size_t};
 
-    pub fn IOServiceClose(a: io_connect_t) -> i32;
+pub(crate) use crate::sys::ffi::*;
 
-    pub fn IOConnectCallStructMethod(
-        connection: mach_port_t,
-        selector: u32,
-        inputStruct: *const KeyData_t,
-        inputStructCnt: size_t,
-        outputStruct: *mut KeyData_t,
-        outputStructCnt: *mut size_t,
-    ) -> i32;
-    // pub fn IORegistryEntryCreateCFProperties(
-    //     entry: io_registry_entry_t,
-    //     properties: *mut CFMutableDictionaryRef,
-    //     allocator: CFAllocatorRef,
-    //     options: IOOptionBits,
-    // ) -> kern_return_t;
-    // pub fn IORegistryEntryGetName(entry: io_registry_entry_t, name: *mut c_char) -> kern_return_t;
+cfg_if! {
+    if #[cfg(not(feature = "apple-sandbox"))] {
+        extern "C" {
```

Comment:
```suggestion
#[cfg(not(feature = "apple-sandbox"))]
extern "C" {
```

No need for `cfg_if` in here. ;)

Reply:
Removed.